quantum_bst_demo.tex

```diff
@@ -0,0 +1,281 @@
+{\small
+\begin{verbatim}
+@article{article_trailing_comma,
+ author={Matthew McKague},
+ doi={10.1017/cbo9780511976667},
+ year={2017},
+ journal={Quantum},
+ volume={1},
+ title={Self-testing in parallel with {CHSH}},
+}
+\end{verbatim}
+}
+The additional comma in the \texttt{title} line raises an internal bibtex warning, which the bibstyle file cannot catch. While in this case no harm is done, there are errors that are not caught but simultaneously lead to incomplete output.
+
+If you want to turn off these errors raised from bibtex warnings, include the entry
+\begin{verbatim*}
+@preamble{"\DoNotMakeWarningsErrors"}
+\end{verbatim*}
+anywhere in the \emph{bibliography} (i.e. the \texttt{.bib} file) like one would include a reference.
+
+\emph{Note:} Commenting the above \texttt{@preamble} entry out via \texttt{\%} will \emph{not} deactivate it as bibtex does not attribute commenting functionality to \texttt{\%}. Instead, you can simply remove the leading \texttt{@}, as is currently done in the \texttt{.bib} file accompanying this document (\texttt{quantum\_bst\_demo.bib}).
+
+\paragraph{Explicit DOI links}
+By default, and if the entry \texttt{doi} is provided for a \texttt{bibitem}, the quantum bibstyle will link to the DOI via the \texttt{journal/volume/issue} part of the citation for articles and via the a) \texttt{volume/chapter/pages} or b) \texttt{number/series} or c) \texttt{publisher} part for books, with a) taking precedence if provided over b) taking precedence if provided over c).
+This behaviour can be changed to output explicit DOI links instead. To do so, include the entry
+\begin{verbatim*}
+@preamble{"\MakeDoiLinksExplicit"}
+\end{verbatim*}
+anywhere in the \emph{bibliography} (i.e. the \texttt{.bib} file) like one would include a reference.
+
+\paragraph{Compression}
+By default, the quantum bibstyle prints the numbers for all references separately and does not compress
+a range of citations, like so: [1, 6, 7, 8, 11, 12, 13, 14].
+This has the advantage of making all references explicitly accessible from the citation position and enables readers to find all occurrences of a given reference.
+If you prefer to compress citations (like so: [1, 6-8, 11-14]), simply add
+\begin{verbatim*}
+\PassOptionsToPackage{compress}{natbib}
+\end{verbatim*}
+to the preamble \emph{before} including
+\begin{verbatim*}
+\usepackage[numbers]{natbib}
+\end{verbatim*}
+where the option \texttt{numbers} is required to make the bibliography style work.
+
+\iffalse
+\section{Tests}\label{sec:tests}
+% \citetests{article_trailing_comma}
+
+Directly from the arxiv~\citetests{mendez2021dissipation}, arxiv via Zotero~\citetests{mendez2021dissipation_2}, some more testcases~\citetests{holevo2012quantum,Holevo_2012,akers2020simple,katariya2021geometric,katariya2021geometric_2,jain2010guaranteed}
+
+Andrea's test case: \citetests{andreas_test_case}.
+
+\bibliographystyletests{quantum}
+\bibliographytests{quantum_bst_demo}
```

Comment:
This is currently hidden in the \iffalse block? For me the current version of the demo does not compile correctly and does not actually contain a bibliography. Can you have another look please, @dwierichs ?

Reply:
Ah, yes but this is only the bibliography for the tests section (there is one bibliography for each section to keep the demo organized).
Maybe these multiple bibliographies are the reason for it not compiling properly on your end? For me it seems to work fine.

Reply:
I will try with a less ancient version of bibtex.